src/Common/src/Interop/Windows/mincore/Interop.Environment.cs

```diff
@@ -16,6 +16,10 @@ internal static unsafe partial class mincore
 
         [DllImport(Libraries.Kernel32, EntryPoint = "ExitProcess")]
         internal static extern void ExitProcess(int exitCode);
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
```

Comment:
This file is included for classic Win32 apps only, but you needs it for all Windows platforms.

This should be added in its own little file instead. You can use https://github.com/dotnet/corert/blob/master/src/Common/src/Interop/Windows/mincore/Interop.GetLastError.cs as template.